### ISSUE TITLE
Initial suggestions

### ISSUE THREAD

**Issue by rolyp** [plrg-bristol/plrg-bristol.github.io#3]
@mengwangoxf would like us to develop a logo, font, colour scheme and (potentially) new name for the PL group at Bristol that we can use on posters/notices/websites etc. I (@rolyp) will be coordinating but everyone is invited to contribute ideas and comments.

### Some guiding principles

- something specific to Bristol in the logo (@lambdabetaeta, @JosephBond)
- not too graphical, mainly words (@mengwangoxf)
- “programming languages” should feature somewhere (@mengwangoxf, @SamFrohlich)

### Mock-ups

From @JosephBond:
![logo idea](https://user-images.githubusercontent.com/121074/196917525-21fb316e-5628-40fb-946d-4591d2d5a1f4.png)

### To do

- [x] make sure everyone is a collaborator on this repo (or org) – currently missing:
  - [x] Joe
  - [x] Hanliang
  - [x] Jess
  - [ ] Jerome
  - [x] Charlie
- [x] name suggestions
- [x] font suggestions
- [x] color scheme suggestions
- [x] logo ideas
- [x] acronym?

**Comment by lambdabetaeta** [comment]
I propose that there must be something distinctly Bristol about it. If not bridge, what?

**Comment by rolyp** [comment]
Banksy?

**Comment by JosephBond** [comment]
Could we overlay a lambda on the clifton suspension bridge

**Comment by ratherforky** [comment]
Meng said he's most concerned about the text, i.e. the name (personally I like just 'Bristol PL'), font, and colour, rather than a graphical logo. That said, I do think it'd be nice to have a graphical logo to go with it. Sam's idea is a lambda suspension bridge, which I quite like

**Comment by SamFrohlich** [comment]
> Banksy?

I like the idea of an edgy graffiti style logo

**Comment by SamFrohlich** [comment]
Should we copy across slack discussion?

**Comment by rolyp** [comment]
@SamFrohlich I’ll do that.

**Comment by SamFrohlich** [comment]
Name suggestion (keep it simple with just):
Bristol Programming Languages
?

**Comment by SamFrohlich** [comment]
Font ideas:
Hero fonts (for titles, the content font should be something easy to read)
graffiti vibes: https://fonts.google.com/specimen/Rock+Salt?preview.text=Bristol%20Programming%20Languages&preview.text_type=custom or https://fonts.google.com/specimen/Rubik+Burned?preview.text=Bristol%20Programming%20Languages&preview.text_type=custom
old video game vibes: https://fonts.google.com/specimen/VT323?preview.text=Bristol%20Programming%20Languages&preview.text_type=custom
(all these are google fonts, which are free use)

**Comment by SamFrohlich** [comment]
> I propose that there must be something distinctly Bristol about it. If not bridge, what?

hot air balloons?

**Comment by SamFrohlich** [comment]
Edgy: 
![image](https://user-images.githubusercontent.com/29566411/196936830-474defb3-1a0f-4407-bdbe-033d29315b2a.png)
(image stolen from google, just getting stuff together a la a mood board)

**Comment by SamFrohlich** [comment]
This is also worth a look https://www.bristol.ac.uk/style-guides/brand-identity/
(gives good examples of how brand guidelines should be, and the sort of things we should and shouldnt be going and included the uni colour schemes inc the main four, and then faculty ones)

**Comment by mengwangoxf** [comment]
I feel that we are on the wrong track. Lambda means something to a very narrow audience, but nothing more. The bridge is a symbol of Bristol, not us. The message we want to send within the uni and externally is that there is a strong programming languages group, and it happens to be in Bristol.

**Comment by rolyp** [comment]
Here’s another mockup based on the following principles:
- use an acronym
- use the word “lab”, make it sounds like we’re real scientists :)
- give some clues to what we’re about
- no bridges, hot air balloons or other Victoriana

Just an example:

<img width="279" alt="Screenshot 2022-10-25 at 08 57 29" src="https://user-images.githubusercontent.com/121074/197716788-09d6004f-9508-44a2-b84c-8f6b9fcb5795.png">

**Comment by SamFrohlich** [comment]
> Here’s another mockup based on the following principles:
> 
> * use an acronym
> * use the word “lab”, make it sounds like we’re real scientists :)
> * give some clues to what we’re about
> * no bridges, hot air balloons or other Victoriana
> 
> Just an example:
> 
> <img alt="Screenshot 2022-10-25 at 08 57 29" width="279" src="https://user-images.githubusercontent.com/121074/197716788-09d6004f-9508-44a2-b84c-8f6b9fcb5795.png">

I like the acronym! Not sure about the layout of the words tho, as I find it hard to parse and see that fmap is the acro. If I get a moment I'll have a bash at organizing those words :-D

**Comment by mengwangoxf** [comment]
You are proposing a complete change of our group's name!
